pgx/src/tupdesc.rs

```diff
@@ -96,17 +97,17 @@ impl<'a> PgTupleDesc<'a> {
     ///
     /// This method is unsafe as we cannot validate that the provided `pg_sys::TupleDesc` is valid
     /// or requires reference counting.
-    pub unsafe fn from_pg_copy<'b>(ptr: pg_sys::TupleDesc) -> PgTupleDesc<'b> {
+    pub unsafe fn from_rust_copy<'b>(ptr: pg_sys::TupleDesc) -> PgTupleDesc<'b> {
```

Comment:
@zombodb can I specifically get your eyes on this bit here? We discussed renaming these at some point and I want confirmation I did the right thing.